packages/styles/components/_c.modal.scss

```diff
@@ -8,7 +8,7 @@
 
   align-items: flex-end;
   display: flex;
-  height: 100vh;
+  height: fill-available;
```

Comment:
In my opinion, we should keep both rules.
Indeed, the `fill-available` property is not supported [everywhere](https://caniuse.com/?search=fill-available).

So, we should keep both rules:

```scss
  height: 100vh;
  height: fill-available;
```
 
 => this way, the most updated browsers will use `fill-available` but the others will use `100vh`.